notebooks/transport/http.go

```diff
@@ -97,19 +108,12 @@ func (h *NotebookHandler) handleGetNotebooks(w http.ResponseWriter, r *http.Requ
 
 // create a single notebook
 func (h *NotebookHandler) handleCreateNotebook(w http.ResponseWriter, r *http.Request) {
-	orgID, err := getIDfromReq(r, "orgID")
```

Comment:
No org ID in the POST API?

Reply:
Not as a URL parameter. We'll get the user info via the authorization on the request context, and eventually will be able to return all of the notebooks they have read access to from the permissions list...hopefully.

Reply:
Hmmmm not much we can do about it I guess, but the inconsistency from our existing APIs worries me. I think OSS is actually ahead of cloud in allowing users to belong to multiple orgs, so we might run into ambiguous situations by deriving the org ID from user permission lists... something to watch out for.